build.sbt

```diff
@@ -1,14 +1,14 @@
 import Dependencies._
 import xerial.sbt.Sonatype._
 
+val mainVersion = "0.3.0-alpha"
+
 lazy val qbeastCore = (project in file("core"))
   .settings(
     name := "qbeast-core",
-    version := "0.1.0",
+    version := mainVersion,
```

Comment:
Are we merging `qbeastCore` and `qbeastSpark` versions?

Reply:
Yes. I believe I made a mistake by not aligning the versions before.
Since we are preparing another release, it is best to solve it here.